packages/maskbook/src/web3/types.ts

```diff
@@ -109,8 +119,16 @@ export type TokenAssetDetailedType<
     T extends EthereumTokenType.ERC721 | EthereumTokenType.ERC1155
 > = TokenAssetDetailedMap[T]
 
-import type { ChainId } from '@dimensiondev/maskbook-shared'
-export { ChainId } from '@dimensiondev/maskbook-shared'
+// Learn more about ethereum ChainId https://github.com/ethereum/EIPs/blob/master/EIPS/eip-155.md
+export enum ChainId {
+    Mainnet = 1,
+    Ropsten = 3,
+    Rinkeby = 4,
+    Gorli = 5,
+    Kovan = 42,
+    Matic = 137,
+    Mumbai = 80001,
+}
```

Comment:
Please add your new definition to `packages\shared\src\web3\index.ts` and revert this change.